app/liquid_tags/asciinema_tag.rb

```diff
@@ -0,0 +1,32 @@
+class AsciinemaTag < LiquidTagBase
+  PARTIAL = "liquids/asciinema".freeze
+
+  def initialize(tag_name, id, tokens)
+    super
+    @id = parse_id(id)
+  end
+
+  def render(_context)
+    ActionController::Base.new.render_to_string(
+      partial: PARTIAL,
+      locals: {
+        id: @id
+      },
+    )
+  end
+
+  private
+
+  def parse_id(input)
+    input_no_space = input.delete(" ")
+    raise StandardError, "Invalid Asciinema Id" unless valid_id?(input_no_space)
+
+    input_no_space
+  end
+
+  def valid_id?(id)
+    id.match?(/\A\d+\Z/i)
```

Comment:
I don't think `/i` is needed since it's only digits, numbers aren't case sensitive :D

Reply:
Hehe, yeah, this tag was mostly a copy of the Blogcast one, which already had the superfluous `i` modifier I didn't notice.